gnome-initial-setup/demo/Makefile.am

```diff
@@ -0,0 +1,19 @@
+
+NULL =
+
+libexec_PROGRAMS = eos-demo-video
+
+eos_demo_video_SOURCES = eos-demo-video.c \
```

Comment:
The point of the `$(NULL)` pattern is to have each file on its own line:

```
foo = \
    bar.c \
    baz.c \
    $(NULL)
```
rather than the last one not having a backslash:

```
foo = \
    bar.c \
    baz.c
```

So either leave it out (since this is one file on one line) or put `eos-demo-video.c` on its own line.